client/src/viewmodel/CreateTrackViewModel.tsx

```diff
@@ -1,20 +1,44 @@
 import { useEffect, useState } from "react";
-import { useParams } from "react-router-dom";
+import { useNavigate, useParams } from "react-router-dom";
 
-import { ProjectResponseType } from "types/projectType";
+import { ProjectResponseType, Track } from "types/projectType";
 
 import { getProject } from "api/project";
+import { addTrack } from "api/track";
 import { trackList as TrackTags } from "utils/data/track";
+import { getAudioBlob } from "utils/audio";
 
 import TrackSetting from "components/blocks/TrackSetting";
 
 function CreateTrackViewModel() {
   const { projectId } = useParams();
+  const navigate = useNavigate();
 
   const [projectInfo, setProjectInfo] = useState<ProjectResponseType>();
   const [isLoading, setIsLoading] = useState(false);
   const isFetched = !isLoading && projectInfo;
 
+  // 트랙 생성하기 버튼 클릭
+  const handleTrackCreate = (title: string, trackTag: Track, audio: HTMLAudioElement) => {
+    if (!projectId) {
+      return;
+    }
+
+    const createTrack = async () => {
+      const recordedBlob = await getAudioBlob(audio);
+
+      const formData = new FormData();
+      formData.append("trackName", title);
+      formData.append("trackTag", trackTag);
+      formData.append("audioFile", recordedBlob);
```

Comment:
인스턴스 생성 구끼리 인접하게 두는게 더 깔끔해보일 것 같아요ㅎㅎ
```suggestion
      const recordedBlob = await getAudioBlob(audio);
      const formData = new FormData();
      
      formData.append("trackName", title);
      formData.append("trackTag", trackTag);
      formData.append("audioFile", recordedBlob);
```

Reply:
저는 코드를 읽을 때 formData를 생성하는 부분이 한 블록으로 읽히면 좋을 것 같아서 묶어뒀어요. 저도 변수 선언부가 인접해있으면 깔끔하다는 의견에 동의해요! 👍🏻